config/jupyterhub_config.py

```diff
@@ -0,0 +1,26 @@
+"""
+This is the JupyterHub configuration file. It is used to configure the JupyterHub server.
+Refer to the JupyterHub documentation for more information:
+https://jupyterhub.readthedocs.io/en/latest/tutorial/getting-started/config-basics.html
+"""
+import os
+
+from jupyterhub_config.custom_spawner import VirtualEnvSpawner
+
+c = get_config()
```

Comment:
Where does the `get_config` name come from? I'd expect this to throw a NameError

Reply:
It's imported automatically by jupyterhub.

Reply:
E.g. it monkey patches the config file?

Reply:
hmmm not exactly sure. But it's ready to use by Jupyterhub automatically. 
https://jupyterhub.readthedocs.io/en/stable/reference/config-reference.html

Reply:
can you link to that in the config as well as the link you already have

Reply:
👍

Reply:
:+1: